disnake/iterators.py

```diff
@@ -535,6 +542,25 @@ async def _retrieve_data(self, retrieve: int) -> AuditLogPayload:
             self.before = Object(id=int(entries[-1]["id"]))
         return data
 
+    async def _after_strategy(self, retrieve: int) -> AuditLogPayload:
+        after = self.after.id
+        data: AuditLogPayload = await self.request(
+            self.guild.id,
+            limit=retrieve,
+            user_id=self.user_id,
+            action_type=self.action_type,
+            after=after,
+        )
+
+        entries = data.get("audit_log_entries", [])
+        if entries:
+            if self.limit is not None:
+                self.limit -= retrieve
+            # note: unlike other paginated endpoints, this one returns entries in ascending
```

Comment:
I wonder if we should be prepared for if/when this reverses randomly...

Reply:
I really hope not. All the other paginated iterators also assume a specific order for pagination - additionally, the order is documented (especially since this one is a somewhat special case):

> When using before, the list is ordered by the audit log entry ID descending (newer entries first). If after is used, the list is reversed and appears in ascending order (older entries first).

\- https://discord.com/developers/docs/resources/audit-log#get-guild-audit-log

Reply:
Let's hope it stays that way.